src/azul/plugins/metadata/hca/aggregate.py

```diff
@@ -80,20 +84,34 @@ def _get_accumulator(self, field) -> Optional[Accumulator]:
             return SetAccumulator(max_size=100)
         elif field in ('size', 'count'):
             return DistinctAccumulator(SumAccumulator())
+        elif field == 'submission_date':
+            return MinAccumulator()
+        elif field == 'update_date':
+            return MaxAccumulator()
         else:
             return None
```

Comment:
This is different to the other implementations of this method. We should create a new _get_default_accumulator() method in the base class and override that here to return `None` instead of `SetAccumulator(max_size=100)`.